fieldsolver/ldz_main.cpp

```diff
@@ -352,9 +352,9 @@ bool propagateFields(
          calculateUpwindedElectricFieldSimple(mpiGrid, sysBoundaries, localCells, RK_ORDER2_STEP1);
          
          propagateMagneticFieldSimple(mpiGrid, sysBoundaries, subcycleDt, localCells, RK_ORDER2_STEP2);
-         // If we are at the first subcycle we need to update the derivatives of the moments, 
-         // otherwise only B changed and those derivatives need to be updated.
-         calculateDerivativesSimple(mpiGrid, sysBoundaries, localCells, RK_ORDER2_STEP2, (subcycleCount==0));
+         // Here, we never need to update the moment derivatives since they will not have changed from
```

Comment:
True, but that doesn't happen yet, so this is, for now, correct.

I would suggest merging this now, if there are no other objections.